libs/designsystem/page/src/page.component.spec.ts

```diff
@@ -793,7 +793,7 @@ describe('PageComponent', () => {
         await TestHelper.whenReady(ionContent);
         const contentInner = ionContent.querySelector('.content-inner');
         expect(contentInner).toHaveComputedStyle({
-          'max-width': '720px',
+          'max-width': DesignTokenHelper.pageContentMaxWidth('default'),
```

Comment:
Used DesignTokenHelper here instead of hardcoded values as I think the unit under test is the pages consistency/alignment with tokens, not the actual value.